scripts/e2e-test.siblings.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Publish with yalc
+yalc publish --sig
+
+# Navigate to siblingsTests and publish without modifying package.json
+cd e2e-test/scripts/siblingsTests/
+
+rm -r .yalc
+rm -r node_modules
+
+cp package.json package.json.bak
+
+yalc add @sentry/capacitor
```

Comment:
We create this folder because it loads faster than pointing yarn to install the root project (10 minutes vs 5 seconds) on the sibling tests